public/API/API_GetAchievementDistribution.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../../lib/bootstrap.php';
+
+runPublicApiMiddleware();
+
+$gameID = requestInputQuery('i');
+$hardcore = requestInputQuery('h', 0, 'integer');
+$requestedBy = requestInputQuery('z');
+$flags = requestInputQuery('f', 3, 'integer');
+
+echo json_encode(getAchievementDistribution($gameId, $hardcore, $requestedBy, $flags));
```

Comment:
You define $gameID but call $gameId which phpstan is complaining about

Reply:
Oops -- should be all set now.